test/quantization/eager/test_quantize_eager_qat.py

```diff
@@ -749,7 +749,10 @@ def forward(self, x):
            use_relu=st.booleans(),
            eps=st.sampled_from([1e-5, 1e-4, 1e-3]),
            momentum=st.sampled_from([0.1, 0.2, 0.3]),
-           freeze_bn=st.booleans())
+           freeze_bn=st.booleans(),
+           zero_gamma=st.booleans(),
+           has_bias=st.booleans(),
+           use_slow_fusion=st.booleans())
```

Comment:
just checking, have we tested the new path with real inputs?  We don't really trust hypothesis anymore to give us realistic inputs and we plan to deprecate it.  I'm assuming you have since you mentioned that various people already use this improvement in their models, but just checking.

Reply:
Yes, we have tested it before. Also if you remove line 817, the test will fail as well.